### PR TITLE
Update Gaia v7

### PR DESCRIPTION
**Pull request by soareschen**
Update to official release Gaia v7.0.0.

### CODE REVIEW

**Comment by soareschen** [comment]
`nix flake check` is failing with various errors. And I can't remember how to call the Nix formatter again.

**Comment by JonathanLorimer** [comment]
I added a script so you should just be able to run `nix develop -c format` now

File: scripts/default.nix [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{pkgs}: let
+  sa = pkgs.writeShellApplication;
+  format = sa {
+    name = "format";
+    text = builtins.readFile ./format.sh;
+    runtimeInputs = with pkgs; [alejandra nix-linter];
+  };
```
Comment (JonathanLorimer) [inline scripts/default.nix:7]: I know this seems like a lot for a trival script where I could probably just use `writeShellScriptBin` but this approach scales better as more scripts are added.

**Review by JonathanLorimer — APPROVED** [review]

**Comment by soareschen** [comment]
> I added a script so you should just be able to run `nix develop -c format` now. Note this runs the formatter and the linter, the linter won't fix the mistakes, it will just report them.

Great, thanks! Is it also documented in the readme? I'll merge this first.

**Pull request merged**